CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
+- A new `FastUVH5Meta` object that enables quick partial reading of uvh5 file metadata.
```

Comment:
We need to detail the updated numpy & scipy requirements in the changelog. That's the thing I check when making conda packages.

Reply:
Also, make sure all your changelog changes end up in the `Unreleased` section after you rebase because we recently released v2.3